docs/run/nodes/nodes.md

```diff
@@ -35,7 +35,7 @@ The js-ceramic node is run as a daemon using Docker or Node.js. By default, Cera
 
 **IPFS Out-of-process**
 
-IPFS can also be in a different process than Ceramic which we refer to as "out-of-process". Running IPFS out-of-process is helpful for more controlled resource allocation, maintenance, debugging, and observability. This is highly recommended if you are planning to be an infrastructure provider for other Ceramic applications.
+The Ceramic daemon can start its own IPFS node internally or it can connect to an externally running IPFS node over HTTP. We refer to the latter as running IPFS "out-of-process". Running IPFS out-of-process is helpful for more controlled resource allocation, maintenance, debugging, and observability. This is highly recommended, especially if you are planning to be an infrastructure provider for other Ceramic applications.
```

Comment:
```suggestion
The Ceramic daemon by default will start its own IPFS node internally, but it can also be configured to connect to an externally running IPFS node over HTTP. We refer to the latter as running IPFS "out-of-process". Running IPFS out-of-process is helpful for more controlled resource allocation, maintenance, debugging, and observability. This is highly recommended, especially if you are planning to be an infrastructure provider for other Ceramic applications.
```